vector/src/main/res/layout/item_text_header.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:paddingVertical="@dimen/layout_vertical_margin"
+    android:paddingHorizontal="@dimen/layout_horizontal_margin">
```

Comment:
`paddingVertical` and `paddingHorizontal` are only available since API 26 and we have to support API 21. I'm surprised that lint does not complain about that. You have to use `paddingStart`, etc.